src/flake8_to_ruff/converter.rs

```diff
@@ -179,6 +179,14 @@ pub fn convert(
                         }
                     }
                 }
+                "allow-untyped-return-if-untyped" | "allow_untyped_return_if_untyped" => {
+                    match parser::parse_bool(value.as_ref()) {
+                        Ok(bool) => flake8_annotations.allow_untyped_return_if_untyped = Some(bool),
+                        Err(e) => {
+                            warn_user!("Unable to parse '{key}' property: {e}");
+                        }
+                    }
+                }
```

Comment:
Generally, we only need to add these if the source plugin defines them. But I don't _think_ this option exists in `flake8-annotations`, does it?

Reply:
Oh sorry, what I meant was: we only need to add this logic to flake8-to-ruff, if it exists in the original plugin. Since, otherwise, we'll never run into them in `.flake8` files!

Reply:
Ah, right, yeah 😄 I based this on `mypy_init_return` and looked for all the places that worked with it. I'll remove this!